.gitignore

```diff
@@ -3,7 +3,7 @@ media/service_image
 media/portfolio_image
 media/doc_images
 static/
-ldap_config.py
```

Comment:
Same as previous comment

Reply:
It's part of the app, not config files. It should nevers contains secrets, as there should be externalized.